inst/ms/aggreCAT.qmd

```diff
@@ -1036,7 +1123,7 @@ different methods and therefore to different assumptions?
 From a computational perspective, multiple aggregation methods must
 first be applied to the forecast prior to comparison and evaluation.
 This can be implemented very succinctly using [purrr]{.pkg}'s
-[map\\_dfr]{.fct} function [@purrr2020] , which takes a list of
+[map\_dfr]{.fct} function [@purrr2020] , which takes a list of
 aggregation methods, repeatedly applies each method to the dataframe
 `focal_claims`, and row-binds the resultant list of dataframes into a
 single dataframe, for example:
```

Comment:
Is this syntax going to be so conveniently "neat" if using multiple types within the same wrapper? E.g. Three AverageWAgg calls, two IntervalWAggs, one BayesianWAgg, and a ReasoningWAgg.

Reply:
Never mind, see that in the subsequent code chunk

Reply:
Although fancy as this approach is, I think it is "beyond" the average user for interpretability. Why not just something like this? Which is essentially what out pipeline does anyway
```
bind_rows(
AverageWAgg(data_ratings,
                       type = "ArMean"),
AverageWAgg(data_ratings,
                       type = "Median"))
```

Reply:
I've changed the first instance of multiple aggregation methods row_bound together, but thought about keeping the other purrr:: approaches given the argument:
> Secondly, the [aggreCAT]{.pkg} package is easily paired with
other tidyverse tools, such as [purrr]{.pkg}, [dplyr]{.pkg}, and
[ggplot2]{.pkg}, as exemplified through the repliCATS workflow described
in @sec-workflow.

I wrote equivalent code as listings instead. But perhaps I could swap this and put the `bind_rows()` approach in main text code chunks and use a single `purrr::` example in a  listing. TBD tomorrow.

Reply:
Yeah, `bind_rows` in text and a `purrr::` listing could be a good solution.